ch2driver/pytpcc/tpcc.py

```diff
@@ -347,10 +356,21 @@ def executorFunc(clientId, TAFlag, driverClass, qDone, warmupDurationQ, warmupDu
         ignore_skip_index_hints = "1"
     os.environ["IGNORE_SKIP_INDEX_HINTS"] = ignore_skip_index_hints
 
+    schema = constants.CH2_DRIVER_SCHEMA["CH2"]
     load_mode = constants.CH2_DRIVER_LOAD_MODE["NOT_SET"]
     bulkload_batch_size = constants.CH2_DRIVER_BULKLOAD_BATCH_SIZE
     kv_timeout = constants.CH2_DRIVER_KV_TIMEOUT
 
+    if args['ch2p']:
+        schema = constants.CH2_DRIVER_SCHEMA["CH2P"]
+    elif args['ch2pp']:
+        schema = constants.CH2_DRIVER_SCHEMA["CH2PP"]
+    if args['customerExtraFields']:
+        customerExtraFields = args['customerExtraFields']
+    if args['ordersExtraFields']:
+        ordersExtraFields = args['ordersExtraFields']
+    if args['itemExtraFields']:
+        itemExtraFields = args['itemExtraFields']
```

Comment:
Right now the variables `customerExtraFields`, `ordersExtraFields` and `itemExtraFields` are only initialized if their corresponding entries in `args` are "truthy".

This means that if you specify any of them as 0 then we get an error like
```
NameError: name 'customerExtraFields' is not defined
```

I don't think we need the `if`s here, we can just do
```
customerExtraFields = args['customerExtraFields']
ordersExtraFields = args['ordersExtraFields']
itemExtraFields = args['itemExtraFields']
```
since we already have default values for them.

Reply:
Fixed.